numalogic/registry/mlflow_registry.py

```diff
@@ -18,6 +18,7 @@
 import mlflow.pyfunc
 import mlflow.pytorch
 import mlflow.sklearn
```

Comment:
I think we don't need these imports?

Reply:
Test cases still need these imports. Otherwise will cause connection errors in tests.

Reply:
Agreed. Let's optimize the imports

Reply:
@yleilawang a good rule of thumb is to make the test cases to work around the actual code. Not the other way round! :)

Reply:
Can we move these imports to test file? Just tried and it didn't cause errors. @ab93 @s0nicboOm

Reply:
Yes test file sounds good!